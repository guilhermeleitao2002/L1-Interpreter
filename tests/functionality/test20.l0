let b = true;
let n = 123;
let result = "Bool: " + b + ", Number: " + n;
result;;