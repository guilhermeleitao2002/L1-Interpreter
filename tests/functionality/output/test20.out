Bool: true, Number: 123